server/resources/resource_management.go

```diff
@@ -888,6 +885,28 @@ func DeleteResource(context middleware.Context,
 	return nil
 }
 
+func fetchResourceForAction(action string, auth schema.Authorization, resourceID string, resourceSchema *schema.Schema,
+	tx transaction.Transaction, context middleware.Context) (*schema.Resource, error) {
+	policy, err := loadPolicy(context, action, strings.Replace(resourceSchema.GetSingleURL(), ":id", resourceID, 1), auth)
+	if err != nil {
+		return nil, err
+	}
+	context["policy"] = policy
```

Comment:
It has already been done in `loadPolicy`

Reply:
good point it is added to context in few places after loadPolicy same as here - i'll remove all such occurrences